### ISSUE TITLE
Debian packaging

### ISSUE THREAD

**Issue by kuba** [kuba/simp_le#69]
For more context (and preferred solution) see https://github.com/kuba/simp_le/pull/68#issuecomment-172066545.

**Comment by athoune** [comment]
Here is package version in Debian Jessie :
- python-cryptography 0.6.1-1
- python-openssl 0.14-1
- no pytz
- python-requests 2.4.3-6

I can try a variant of #68

**Comment by alanivey** [comment]
@kuba I have documented how to create non-virtualenv packages for CentOS 6 and 7. If you'd like me to add them to a Wiki page, let me know and I'll be happy to share.

**Comment by kuba** [comment]
That's great, please do share!

BTW, any chance for reproducible scripts that could be pulled directly to
the the repo (PR)? :)

**Comment by alanivey** [comment]
Sure, I'll start another issue so as to not hijack this one b/c I do have a question.

**Comment by bit** [comment]
Once #83 is merged, you can create a valid debian package with the following 2 lines:

```
python3 setup.py --command-packages=stdeb.command debianize
echo "acme python3-acme (>= 0.4.0)" > debian/py3dist-overrides
```

**Comment by athoune** [comment]
I need a package for Debian Wheezy, so I keep my dh-virtualenv hack.

But it's great if an official package is available for contemporary Debian.

**Comment by kuba** [comment]
#83 was merged, as a part of #98.

@bit, do you mean `debianize` that is part of https://pypi.python.org/pypi/stdeb?

**Comment by bit** [comment]
>  do you mean debianize that is part of https://pypi.python.org/pypi/stdeb?

@kuba yes debianize is part of stdeb (apt install python3-stdeb)
